lib/sidekiq/retry_monitoring.rb

```diff
@@ -13,6 +13,8 @@ def call(worker, params, _queue)
     private
 
     def should_notify?(worker, job)
+      return false unless job['retry_count']
```

Comment:
@yakschuss / @LindseySaari Do you think that this is part of the problem I'm seeing here - https://github.com/department-of-veterans-affairs/va.gov-team/issues/26326